gnpy/core/elements.py

```diff
@@ -246,6 +247,7 @@ def __str__(self):
             return f'{type(self).__name__} {self.uid}'
 
         return '\n'.join([f'{type(self).__name__} {self.uid}',
+                          f'  type_variety:      {self.config.type_variety}',
```

Comment:
Is it `self.type_variety` or `self.config.type_variety`?

Reply:
the code works...

Reply:
James, 
the code works...